django-backend/fecfiler/authentication/views.py

```diff
@@ -37,39 +51,136 @@ class AccountViewSet(GenericViewSet, ListModelMixin):
         "email",
         "role",
         "is_active",
-        "name"
+        "name",
     ]
     ordering = ["name"]
 
     def get_queryset(self):
-        queryset = Account.objects.annotate(
-            name=Concat('last_name', Value(', '), 'first_name', output_field=CharField())
-        ).filter(cmtee_id=self.request.user.cmtee_id).all()
+        queryset = (
+            Account.objects.annotate(
+                name=Concat(
+                    "last_name", Value(", "), "first_name", output_field=CharField()
+                )
+            )
+            .filter(cmtee_id=self.request.user.cmtee_id)
+            .all()
+        )
 
         return queryset
 
 
 class LoginDotGovSuccessSpaRedirect(View):
```

Comment:
What do we get from having this in a separate class, rather than making it an action in the AccountViewSet.  Same with `LoginDotGovSuccessLogoutSpaRedirect`, `LogoutView`, and `authenticate_login` (a separate api_view rather than a class)

Reply:
https://www.django-rest-framework.org/api-guide/viewsets/#viewset

Reply:
Will make this refactor in a later PR.